src/tokenserver/extractors.rs

```diff
@@ -72,62 +69,73 @@ impl TokenserverRequest {
     /// of the FxA server may not have been sending all the expected fields, and
     /// that some clients do not report the `generation` timestamp.
     fn validate(&self) -> Result<(), TokenserverError> {
+        /// Returns true only if both arguments are Some and opta > optb.
+        fn gt(opta: Option<i64>, optb: Option<i64>) -> bool {
```

Comment:
Open to suggestions here...I wanted to just compare the options themselves using comparison operators, but the semantics of `Option`'s impl for `PartialOrd` didn't work for my use case: https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=2073cf8cbebae1806f073b04c3110dd2

Reply:
I tried my hand at writing my own (very simple) `opt_cmp!` macro to accomplish something similar, but IMO it's a bit more readable. It lets you do something like `opt_cmp!(opta < optb)`, where `opta` and `optb` are both `Option`s. Let me know what you think!